packages/chart/src/plotly/Plotly.js

```diff
@@ -3,14 +3,13 @@
 // https://github.com/plotly/plotly.js/#modules
 
 import Plotly from 'plotly.js/lib/core.js';
+import bar from 'plotly.js/lib/bar.js';
+import histogram from 'plotly.js/lib/histogram.js';
+import pie from 'plotly.js/lib/pie.js';
+import ohlc from 'plotly.js/lib/ohlc.js';
+import scattergl from 'plotly.js/lib/scattergl.js';
 
 // Load in the trace types we need/support
-Plotly.register([
-  require('plotly.js/lib/bar'),
-  require('plotly.js/lib/histogram'),
-  require('plotly.js/lib/pie'),
-  require('plotly.js/lib/ohlc'),
-  require('plotly.js/lib/scattergl'),
-]);
+Plotly.register([bar, histogram, pie, ohlc, scattergl]);
```

Comment:
Just curious why change to import?

Reply:
It was invalid ESM before. Can't use `require` within an ESM and we're publishing as ESM. Webpack magically makes the old way work